products/cert-manager.md

```diff
@@ -9,6 +9,10 @@ releasePolicyLink: https://cert-manager.io/docs/releases/
 changelogTemplate: https://github.com/cert-manager/cert-manager/releases/tag/v__LATEST__
 releaseDateColumn: true
 eolColumn: Support
+versionCommand: |-
```

Comment:
Must be moved before the `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).